BurnerB/FreeMentors-V2
# #168402386 Implement Lfa feedback

#### What does this PR do?
Impement lfa feedback
#### Description of Task to be completed?
* Make all error variables the same
* hardcode tokens
#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/FreeMentors-V2.git and cd into FreeMentors-V2
* checkout to branch ch-DB-implement-feedback-168402386
* Run npm --init to install all dependancies.
* Run command `npm test` on your terminal
* Tests should pass
* 
#### What are the relevant pivotal tracker stories?
[#168402386](https://www.pivotaltracker.com/story/show/168402386)
#### Screenshots (if appropriate)
